test/fun_prop.lean

```diff
@@ -59,7 +61,8 @@ or
 ```
 Measurable.fst : Measurable f → Measurable fun x => Prod.fst (f x)
 ```
-Tactic `fun_prop` can work with both versions and it should be sufficient to provide just one of them.
+Tactic `fun_prop` can work with both versions and it should be sufficient to provide just one of
```

Comment:
How about
"The tactic `fun_prop` can work with both versions; it should be sufficient to provide just one of them."?

Reply:
Thanks: changed!